src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -4270,10 +4307,17 @@ namespace ts.projectSystem {
                 verifyProjectAndWatchedDirectories();
                 callsTrackingHost.verifyNoHostCalls();
 
+                function getFilePathIfOpen(f: FileOrFolder) {
+                    const path = toCanonical(f.path);
+                    const info = projectService.getScriptInfoForPath(toCanonical(f.path));
+                    return info && info.isScriptOpen() ? undefined : path;

```

Comment:
This seems to do the opposite of what it's named as? If the file is open, it *doesn't* return the path?